pandas/tools/plotting.py

```diff
@@ -1,3 +1,21 @@
-# flake8: noqa
+import sys
+import warnings
 
-from pandas.plotting.api import *
+import pandas.plotting.api as api
+
+# back-compat of public API
+# deprecate these functions
+m = sys.modules['pandas.tools.plotting']
+for t in [t for t in dir(api) if not t.startswith('_')]:
+
+    def outer(t=t):
+
+        def wrapper(*args, **kwargs):
+            warnings.warn("pandas.tools.plotting{t} is deprecated. "
+                          "import from the "
+                          "pandas.plotting.{t} instead".format(t=t),
```

Comment:
I think the "from the" can be left out.

I would do either: 
"pandas.tools.plotting.scatter_matrix is deprecated, import pandas.plotting.scatter_matrix instead" 
or 
"pandas.tools.plotting.scatter_matrix is deprecated, import from pandas.plotting instead"